From: Sebastian Ramacher <sramacher@debian.org>
Date: Sat, 3 Sep 2022 19:30:13 +0200
Subject: Do not build unused blake2 dependency

---
 deps/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/deps/CMakeLists.txt b/deps/CMakeLists.txt
index 9e92bb1..992a51d 100644
--- a/deps/CMakeLists.txt
+++ b/deps/CMakeLists.txt
@@ -9,7 +9,6 @@ if(OS_WINDOWS)
   add_subdirectory(ipc-util)
 endif()
 
-add_subdirectory(blake2)
 add_subdirectory(glad)
 add_subdirectory(media-playback)
 add_subdirectory(file-updater)
